djstripe/templates/djstripe/account.html

```diff
@@ -27,15 +27,6 @@
 
 
 
```

Comment:
we switched to model plans. attribute names are different. I will try to describe in more detail each condition